### DISCUSSION TITLE
Error: binary  missing

### DISCUSSION THREAD

**Discussion by cancundeyingzi** [bmax121/APatch#453] [Q&A]
![Screenshot_2024-05-18-08-59-31-194_me bmax apatch](https://github.com/bmax121/APatch/assets/73635883/2d22cd88-c0f6-4ccf-b478-f5935769fbd3)
请问刷入任意模块都显示这个错误需要如何处理？已在各大引擎搜索过没有解决方法。

**Comment by pomelohan** [comment]
走 issue 渠道反馈。
